INSTALL

```diff
@@ -591,6 +593,17 @@ The specific libraries and their options are:
   URL: https://zlib.net
   Options: ZLIB_OPT, ZLIB_LIB.
 
+- GSLib, used when MFEM_USE_GSLIB = YES. GSLib must be built prior to the 
+  MFEM build.
+  URL: https://github.com/gslib/gslib/archive/v1.0.5.tar.gz
```

Comment:
No, we are not fixed to this particular release. v1.0.6 is about to come out soon and it should be compatible with MFEM.

Reply:
On second thought, since backwards compatibility is not guaranteed, it will be better to specify the version number. I will leave it specified to v1.0.5 for now.